package.json

```diff
@@ -137,7 +137,7 @@
     "ngx-bootstrap": "3.0.1",
     "ngx-fabric8-wit": "8.2.0",
     "ngx-feature-flag": "1.0.0",
-    "ngx-launcher": "2.0.9",
+    "ngx-launcher": "3.0.11",
```

Comment:
```suggestion
    "ngx-launcher": "3.0.13",
```

Reply:
Don't you also need to update the `package-lock.json`?